packages/system/src/MasterList/api/operations.graphql

```diff
@@ -1,3 +1,23 @@
+fragment MasterListLine on MasterListLineNode {
+  id
+  item {
+    ...ItemRow
```

Comment:
This causes a bit of a funky import from the `Item` package. Since these aren't tree shaking properly right now, it doesn't make a difference but would cause the Item bundle to be bundled with this package even if shaken? Probably a later concern

Reply:
ah. yes - sorry. forgot about that - I edited the generated operations file and meant to go back and just duplicate the `ItemRow` instead.